Build/start.sh

```diff
@@ -28,9 +27,10 @@ echo "    CONFIGURATION COPIED TO conf/"
 if [ -z $DLOAD ];
 then
 	cd $WORK/build
-	7z x ~/oss_pkg_v3.0.0.7z
+	7z x ~/oss_pkg_rzv_v3.0.0.7z
```

Comment:
Is there a reason that you are using the `rzv` instead?

Reply:
WIP